packages/forms/stories/json/fields/core-nestedListView.json

```diff
@@ -50,14 +50,17 @@
             "carot": "Carot",
             "lettuce": "Lettuce",
             "turnip": "Turnip"
-          }
+          },
+          "checked": true
```

Comment:
I don't understand the story behind sorry.
The option should be enough.
if option and items.length >0 => expand.
Also checked in uischema do not look like an item data.